node.go

```diff
@@ -246,6 +250,30 @@ func (n *node) collectedCount() (count int64) {
 	return
 }
 
+func (n *node) emittedCount() (count int64) {
+	for _, out := range n.outs {
+		count += out.collectedCount()
```

Comment:
Just to make sure:
`node collected count is the sum of emitted counts of parent edges` above collectedCount 
`node emitted count is the sum of collected counts of children edges` above emittedCount

?